docs/platform/howto/manage-org-users.rst

```diff
@@ -0,0 +1,59 @@
+Manage users in an organization
+================================
+
+.. important::
+    Organization users are available as a :doc:`feature preview </docs/platform/howto/feature-preview>`.
```

Comment:
Can we add a sentence that the feature preview has enabled? Just to avoid support tickets that the menu item is not visible.